### ISSUE TITLE
Error when activation is specified in MLPEncoder

### ISSUE THREAD

**Issue by yukw777** [asappresearch/flambe#65]
**Describe the bug**
When I specify activation layers for the MLPEncoder, I see the following error:
```
2019-09-03 11:51:52,138	ERROR trial_runner.py:487 -- Error processing event.
Traceback (most recent call last):
  File "/home/peter/code/flambe/venv/lib/python3.6/site-packages/ray/tune/trial_runner.py", line 436, in _process_trial
    result = self.trial_executor.fetch_result(trial)
  File "/home/peter/code/flambe/venv/lib/python3.6/site-packages/ray/tune/ray_trial_executor.py", line 323, in fetch_result
    result = ray.get(trial_future[0])
  File "/home/peter/code/flambe/venv/lib/python3.6/site-packages/ray/worker.py", line 2195, in get
    raise value
ray.exceptions.RayTaskError: ray_worker (pid=9937, host=peter-MS-7758)
  File "/home/peter/code/flambe/venv/lib/python3.6/site-packages/ray/tune/trainable.py", line 87, in __init__
    self._setup(copy.deepcopy(self.config))
  File "/home/peter/code/flambe/flambe/experiment/tune_adapter.py", line 76, in _setup
    block.load_state(state)
  File "/home/peter/code/flambe/flambe/compile/component.py", line 1093, in load_state
    load(self)
  File "/home/peter/code/flambe/flambe/compile/component.py", line 1092, in load
    load(child, prefix + name + STATE_DICT_DELIMETER)
  File "/home/peter/code/flambe/flambe/compile/component.py", line 1092, in load
    load(child, prefix + name + STATE_DICT_DELIMETER)
  File "/home/peter/code/flambe/flambe/compile/component.py", line 1092, in load
    load(child, prefix + name + STATE_DICT_DELIMETER)
  [Previous line repeated 1 more time]
  File "/home/peter/code/flambe/flambe/compile/component.py", line 1083, in load
    unexpected_keys, error_msgs)
  File "/home/peter/code/flambe/venv/lib/python3.6/site-packages/torch/nn/modules/module.py", line 685, in _load_from_state_dict
    hook(state_dict, prefix, local_metadata, strict, missing_keys, unexpected_keys, error_msgs)
  File "/home/peter/code/flambe/flambe/compile/component.py", line 977, in _load_state_dict_hook
    version = local_metadata[VERSION_KEY].split('.')
KeyError: '_flambe_version'
```

**To Reproduce**
Use the following yaml file:
```yaml
!Experiment

name: sst-text-classification

pipeline:

  # stage 0 - Load the Stanford Sentiment Treebank dataset and run preprocessing
  dataset: !SSTDataset
    transform:
      text: !TextField
      label: !LabelField

  # Stage 1 - Define a model
  model: !TextClassifier
    embedder: !Embedder
      embedding: !torch.Embedding  # automatically use pytorch classes
        num_embeddings: !@ dataset.text.vocab_size
        embedding_dim: 300
      embedding_dropout: 0.3
      encoder: !MLPEncoder
        input_size: 300
        output_size: 128
        output_activation: !torch.ReLU
        n_layers: 2
        hidden_size: 256
        hidden_activation: !torch.ReLU
    output_layer: !SoftmaxLayer
        input_size: 128
        output_size: !@ dataset.label.vocab_size

  # Stage 2 - Train the model on the dataset
  train: !Trainer
    dataset: !@ dataset
    model: !@ model
    train_sampler: !BaseSampler
    val_sampler: !BaseSampler
    loss_fn: !torch.NLLLoss
    metric_fn: !Accuracy
    optimizer: !torch.Adam
      params: !@ train.model.trainable_params
    max_steps: 10
    iter_per_step: 100

  # Stage 3 - Eval on the test set
  eval: !Evaluator
    dataset: !@ dataset
    model: !@ train.model
    metric_fn: !Accuracy
    eval_sampler: !BaseSampler

# Define how to schedule variants
schedulers:
  train: !tune.HyperBandScheduler
```
1. run the experiment.
1. you'll see the error.

**Expected behavior**
No error should be raised.

**Screenshots**

**Software Versions (please complete the following information):**
 - OS: Ubuntu 18.04
 - Python Version 3.6.8
 - PyTorch Version 1.1.0
 - Flambé Version master

**Additional context**

**Comment by jeremyasapp** [comment]
@nmatthews-asapp is in the process of addressing this

**Comment by jeremyasapp** [comment]
Fixed by #71
